src/main/java/bio/terra/service/snapshotbuilder/SnapshotBuilderService.java

```diff
@@ -75,53 +76,55 @@ public SnapshotAccessRequestResponse createSnapshotRequest(
   }
 
   private <T> List<T> runSnapshotBuilderQuery(
-      String sql,
+      Query query,
```

Comment:
Why change runSnapshotBuilderQuery to take the query instead of the string? Is it so the context doesn't need to be passed around- it can just be created here?